docs/contributing/setting-up-your-local-dev-environment.md

```diff
@@ -21,6 +21,7 @@ Yarn is a package manager for your code, similar to [NPM](https://www.npmjs.com/
 
 - [Install](https://yarnpkg.com/en/docs/install) the Yarn package manager.
 - Ensure you have the latest version of Yarn installed (>= 1.0.2). `yarn --version`
+- Ensure you have the latest LTS version of Node installed (>= 10.16). `node --version`
```

Comment:
I didn't realize this PR was open and was addressing https://github.com/gatsbyjs/gatsby/issues/18550 issue (no reference to that issue in this PR), and I went ahead and merged https://github.com/gatsbyjs/gatsby/pull/18551. So let's remove this change as we would end up with 2 exactly same lines.

There's still valuable change about `.env.development` file in this PR, so we still will merge that after this change